gradle/publish.gradle

```diff
@@ -41,9 +41,11 @@ publishing {
 
 nexusPublishing {
     repositories {
+        // TODO: the following line causes the `The Project.getConvention() method has been deprecated.` warning.
```

Comment:
The `gradle-nexus` plugin still is on [gradle 8.2.1](https://github.com/gradle-nexus/publish-plugin/blob/c2614e3a5fd61008c66633ca061760df8d4a5106/gradle/wrapper/gradle-wrapper.properties#L4).
I tracked down the deprecation warning to the `sonatype` block, but didn't find a way to avoid the warning.

Reply:
Let's note that in the comment so we're not tempted to track it down again. It's not really a TODO until they fix the plugin.